sdk/java-sdk/src/main/scala/com/akkaserverless/javasdk/impl/Serializer.scala

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.akkaserverless.javasdk.impl
+
+import com.akkaserverless.javasdk.JsonSupport
+// FIXME: should we duplicate this in the scala sdk and expose ScalaPbAny instead?
+import com.google.protobuf.{ Any => JavaPbAny }
+
+object Serializer {
+
+  def noopSerializer: Serializer = NoopSerializer
+
+  object NoopSerializer extends Serializer {
+    override def canSerialize(any: Any): Boolean = false
+
+    override def serialize(any: Any): JavaPbAny =
+      throw new IllegalArgumentException("NoopSerializer can not serialize types")
+
+    override def canDeserialize(any: JavaPbAny): Boolean = false
+
+    override def deserialize(any: JavaPbAny): Any =
+      throw new IllegalArgumentException("NoopSerializer can not deserialize types")
+
+  }
+
+}
+trait Serializer {
+  def canSerialize(any: Any): Boolean
+  def serialize(any: Any): JavaPbAny
+
+  def canDeserialize(any: JavaPbAny): Boolean
+  def deserialize(any: JavaPbAny): Any
```

Comment:
Could have define to types, but then we will have to provide `Ser` and `Deser` while there isn't a case where only one should be provided.